docs/user-guide/src/bmo/live-iso.md

```diff
@@ -0,0 +1,40 @@
+# Live-ISO
+
+The live-iso API in Metal3 allows to boot BaremetalHost with live-iso image instead of using IPA deploy ramdisk.
+
+Why we need it?
+
+In some circumstances, i.e to reduce boot time for ephemeral workloads, it may be possible to boot an iso
+and not deploy any image to disk (saving the time to write the image and reboot). Since this alternative
+is distributed as a live-iso, leveraging the existing toolchains like [fedora-coreos installer](https://docs.fedoraproject.org/en-US/fedora-coreos/bare-metal/) might be desirable. On top that, Ironic already exposes an [API](https://docs.openstack.org/ironic/latest/admin/ramdisk-boot.html) to support this.
```

Comment:
I would skip the last sentence here since it is already obvious that Ironic must support it if BMO does. Or we could say something like "this is implemented using the ramdisk-boot API in Ironic" so the user can read more about it if interested.
```suggestion
is distributed as a live-iso, leveraging the existing toolchains like [fedora-coreos installer](https://docs.fedoraproject.org/en-US/fedora-coreos/bare-metal/) might be desirable.
```

Reply:
I got rid of it, thanks